sources/packages/backend/libs/sims-db/src/entities/student-loan-balances.model.ts

```diff
@@ -0,0 +1,42 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+import { numericTransformer } from "../transformers/numeric.transformer";
+
+/**
+ * Stores files information and content from NSLSC
+ * Loan Balance file.
+ */
+@Entity({ name: TableNames.StudentLoanBalances })
+export class StudentLoanBalances extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * Student id associated with this Loan Balance.
+   */
+  @Column({
+    name: "student_id",
+    type: "integer",
+    nullable: false,
+  })
+  studentId: number;
```

Comment:
We generally add the relationship as ORM model(ManyToOne, OneToMany etc..) instead of just a regular column reference. 

Is there a need here to add like this?

Reply:
The below comment explained and asked for this property to be removed. How did we end up here?
https://github.com/bcgov/SIMS/pull/3036#discussion_r1543848299